### ISSUE TITLE
Release update with pem-0.7

### ISSUE THREAD

**Issue by djc** [est31/rcgen#27]
Would be nice to have a release out that doesn't (transitively) depend on base64-0.10.

**Comment by est31** [comment]
It has to be 0.8.0 because of pem's error type being part of the API.

**Comment by djc** [comment]
Thanks!

**Comment by est31** [comment]
(Release was in 8106ff63fd94f9c3be8adebb9cecb07d15cdfcf2 )
